status/projects.py

```diff
@@ -459,7 +467,12 @@ class RunningNotesDataHandler(SafeHandler):
     def get(self, project):
         self.set_header("Content-type", "application/json")
         p = Project(lims, id=project)
-        p.get(force=True)
+        try:
+            p.get(force=True)
+        except: # Don't want to create dependency on urllib2 just to catch a HTTPError
+            raise tornado.web.HTTPError(404, reason='Project not found: {}'.format(project))
+            # self.set_status(404)
+            # self.write('{}')
```

Comment:
Not big problems - just that it was returning a `500` status because of an uncaught exception instead of `404`. This should handle it in a better way.